vo-dml/desc/desc.error.Ellipse.txt

```diff
@@ -1,3 +1,3 @@
 Elliptic error for 2D parameters such as sky positions.
 Major axis and minor axis have their own units, which must be the same for both.
-This is not enforced by the model.
\ No newline at end of file
+This is not enforced by the model. The definition of the ellipse attribute is imported from \texttt{coords:Ellipse}. 
```

Comment:
1) I think the definitions match only for the standard a celestial coordinate space (see comments on issue #60.
2) Ellipse is in Measurements, so is meas:Ellipse